src/main.rs

```diff
@@ -1,3 +1,17 @@
+use std::fs;
+use std::path::PathBuf;
+use std::env;
+
 fn main() {
```

Comment:
Make this return a Result with a boxed error. Then remove all the error handling and just propagate them